modulemd/v2/include/modulemd-2.0/modulemd-component.h

```diff
@@ -43,6 +43,18 @@ struct _ModulemdComponentClass
   gpointer padding[7];
 };
 
+/**
+ * modulemd_component_equals_wrapper:
+ * @a: A void pointer
+ * @b: A void pointer
+ *
+ * Returns: TRUE, if both the pointers are equal. FALSE, otherwise
+ *
+ * Since: 2.3
+ */
+gboolean
+modulemd_component_equals_wrapper (const void *a, const void *b);
```

Comment:
Do not make these wrappers public. They aren't meaningful for the external API. If you need them in multiple places (e.g. both ModuleStream objects), then these belong in the private headers like `modulemd/v2/private/modulemd-component-private.h`)